frontend/src/components/Categories/Categories.tsx

```diff
@@ -0,0 +1,28 @@
+import { ChangeEvent, useState } from 'react';
+import './styles.css';
+
+const Categories = (props: {
```

Comment:
consider renaming. What does this component do exactly? if it's menu let's call it menu

Reply:
renamed to Menu